XamarinCommunityToolkit/Behaviors/TagType.shared.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Behaviors
+{
+    public class TagType
```

Comment:
Just a question @rdelrosario  @sthewissen @jfversluis 

Do you, folks, think it can be useful to make this class inherited from BindableObject with making its properties as bindable?

We can pass bindingcontext from the View

Reply:
I'm not sure about this, I think it would make sense if it was using within a DataTemplate but it isn't